.github/workflows/CheckBadMerge.groovy

```diff
@@ -34,9 +34,10 @@ import java.nio.file.Paths
 class CheckBadMerge {
     private static final THREAD_POOL = Executors.newCachedThreadPool()
 
-    private static final List<String> MONITORED_FILES = [
+    private static final List<String> MONITORED_PATHS = [
         "subprojects/docs/src/docs/release/notes.md",
         "platforms/documentation/docs/src/docs/release/notes.md",
+        "platforms/documentation/docs/src/docs/release/release-notes-assets",
```

Comment:
💅 we could have this path end in a `/` so it can only ever match subfolders or files below that relative path instead of also matching `.../release-notes-assets-something.txt`,...